streams/writable-streams/aborting.js

```diff
@@ -694,8 +694,8 @@ promise_test(t => {
     ]);
   }).then(() => {
     assert_array_equals(
-        events, [],
-        'writePromise, abortPromise and writer.closed must not be fulfilled/rejected yet even after ' +
+        events, ['abortPromise'],
```

Comment:
I think promise_rejects(... ,abortPromise) should move from line 705 to line 689 now.